qds_sdk/account.py

```diff
@@ -43,6 +43,15 @@ def parsers():
             choices=["true", "false"], default="false",
             help="Use previous account plan, default: false")
         create.set_defaults(func=AccountCmdLine.create)
+
+        #branding
+        branding = subparsers.add_parser(
```

Comment:
Confirmation from API perspective.
1. This API overwrites the logo--uri in case there already exist one for a given account? 
2. All the three links/uri are optional. Making a blank call with just account-id will have no effect. 
3. What is the label of documentation URL?